src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -1127,20 +1145,20 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 
 		// Build labels and descriptions for each editor
 		const labels: IEditorInputLabel[] = [];
-		let activeEditorIndex = -1;
-		for (let i = 0; i < this.group.editors.length; i++) {
-			const editor = this.group.editors[i];
+		let activeEditorTabIndex = -1;
+		for (let i = 0; i < this.tabsModel.getEditors(EditorsOrder.SEQUENTIAL).length; i++) {
+			const editor = this.tabsModel.getEditors(EditorsOrder.SEQUENTIAL)[i];
 			labels.push({
```

Comment:
This seems not very efficient. You call `getEditors` in a loop.

Reply:
This is how it used to be implemented, it just was hidden behind a getter method, changed it